app/Services/PivotalTrackerClient.php

```diff
@@ -105,25 +106,72 @@ public function loadStoriesForProject(): array
             ]
         ]);
 
-        return $this->filterByReviews(json_decode($response->getBody(), true));
+        return $this->filterStories(json_decode($response->getBody(), true));
     }
 
     /**
      * Loads all reviews of a certain story
      *
-     * @param int $projectId
      * @param int $storyId
      * @return array
      * @throws GuzzleException
      */
-    public function loadReviewsForStory(int $projectId, int $storyId): array
+    public function loadReviewsForStory(int $storyId): array
     {
-        $response = $this->client->request('GET', '/services/v5/projects/' . $projectId . '/stories/' . $storyId . '/reviews', [
+        $response = $this->client->request('GET', '/services/v5/projects/' . config('pivotal-tracker.project_id') . '/stories/' . $storyId . '/reviews', [
             'headers' => [
                 'X-TrackerToken' => config('pivotal-tracker.api_token'),
             ]
         ]);
 
         return json_decode($response->getBody(), true);
     }
+
+    /**
+     * Checks if the set review type is set to pass
+     *
+     * @param array $reviews
+     * @return bool
+     */
+    private function filterByReviews(array $reviews): bool
+    {
+        foreach ($reviews as $review) {
+            if ($review['review_type_id'] == config('pivotal-tracker.review_type') && $review['status'] == 'pass') {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * Loads and returns the message sent when setting newsletter review to pass
+     *
+     * @param int $storyId
+     * @return string
+     * @throws GuzzleException
+     */
+    private function loadNewsletterMessage(int $storyId): string
+    {
+        $response = $this->client->request('GET', '/services/v5/projects/' . config('pivotal-tracker.project_id')  . '/stories/' . $storyId . '/comments', [
+            'headers' => [
+                'X-TrackerToken' => config('pivotal-tracker.api_token'),
+            ]
+        ]);
+
+        $comments = json_decode($response->getBody(), true);
+
+        foreach($comments as $comment) {
+            if(str_contains($comment['text'], '**Newsletter** review set to **pass**')) {
+                $exploded = preg_split('/\r\n|\r|\n/', trim($comment['text']));
+
+                if(isset($exploded[2])) {
+                    return $exploded[2];
+                }
+                return 'There is no pass message!';
+            }
+        }
+
+        throw new \Exception('Something went wrong when loading comments');
```

Comment:
You should just return here empty string instead. It's not wrong if there is not newsletter text ;)